docs/src/main/asciidoc/deploying-to-openshift.adoc

```diff
@@ -1,281 +1,420 @@
+:RHEL: Red Hat Enterprise Linux
+:RHELVersion: 8
 
+//Quarkus URLs
 
-== Prerequisites
+:URL_COMPONENT_FORMAT: html-single
+:URL_COMPONENT_PRODUCT: red_hat_build_of_quarkus
+:URL_BASE_QUARKUS: https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/{ProductVersion}/{URL_COMPONENT_FORMAT}
 
-To complete this guide, you need:
+:GETTING_STARTED_QUARKUS: Getting started with {PRODUCTLONGNAME}
+:RELEASE_NOTES_QUARKUS: {PRODUCTLONGNAME} release notes
+:MAVEN_QUARKUS: Creating {PRODUCTNAME} applications with Apache Maven
+:OPENSHIFT_QUARKUS: Deploying {PRODUCTNAME} applications on {RHOSLong}
+:CONFIGURATION_QUARKUS: Configuring {ProductName} applications
+:TESTING_QUARKUS: Testing your {ProductName} Getting Started project
+:NATIVE_EXECUTABLE_QUARKUS: Building a native executable with {PRODUCTLONGNAME}
 
-* roughly 5 minutes
-* an IDE
-* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
-* Apache Maven {maven-version}
-* access to an OpenShift cluster (Minishift is a viable option)
-* OpenShift CLI (Optional. Only required for manually deploying)
+:URL_RELEASE_NOTES: {URL_BASE_QUARKUS}/release_notes_for_{URL_COMPONENT_PRODUCT}_{ProductVersion}
+:URL_GETTING_STARTED_QUARKUS: {URL_BASE_QUARKUS}/getting_started_with_{URL_COMPONENT_PRODUCT}
+:URL_MAVEN_QUARKUS: {URL_BASE_QUARKUS}/creating_quarkus_applications_with_apache_maven
+:URL_OPENSHIFT_QUARKUS: {URL_BASE_QUARKUS}/deploying_quarkus_applications_on_red_hat_openshift_container_platform
+:URL_CONFIGURATION_QUARKUS: {URL_BASE_QUARKUS}/configuring_quarkus_applications
+:URL_TESTING_QUARKUS: {URL_BASE_QUARKUS}/testing_your_quarkus_getting_started_project
+:URL_NATIVE_EXECUTABLE_QUARKUS: {URL_BASE_QUARKUS}/building_a_native_executable_with_{URL_COMPONENT_PRODUCT}
 
-== Creating the Maven project
+//External URLs
+:URL_OPENSHIFT_INSTALLING: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/installing/index
 
-First, we need a new project that contains the OpenShift extension. This can be done using the following command:
+= Deploying Quarkus applications on {RHOSLong}
 
-[source,bash,subs=attributes+]
-----
-mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
-    -DprojectGroupId=org.acme \
-    -DprojectArtifactId=openshift-quickstart \
-    -DclassName="org.acme.rest.GreetingResource" \
-    -Dpath="/greeting" \
-    -Dextensions="openshift"
+:context: quarkus-openshift
+//include::_artifacts/author-group.adoc[]
 
-cd openshift-quickstart
-----
 
-=== OpenShift
+As an application developer, you can deploy a {ProductName} application on {RHOSLong} using Apache Maven and the `quarkus-openshift` extension in a single build command or you can use the traditional source-to-image (S2I) method. The resulting image from either method is fully supported. See the https://access.redhat.com/support/offerings/developer/soc/[Development Support Scope of Coverage] page for the build process and tools that are covered under development support.
```

Comment:
I am sure this has been answered before, but what does it mean from a community perspective that `The resulting image from either method is fully supported`?